config/export.yaml

```diff
@@ -1256,6 +1262,11 @@ data:
   children:
   - 2.2i/calib/gen2
   - LSSTCam-imSim/calib/curated
+- type: collection
+  collection_type: CHAINED
+  name: models
```

Comment:
```
                    Name                         Type
-------------------------------------------- -----------
2.2i/calib/gen2                              CALIBRATION
2.2i/calib/gen2/20220101T000000Z             RUN
2.2i/calib/gen2/20220806T000000Z             RUN
LSSTCam-imSim/calib                          CHAINED
  2.2i/calib/gen2                            CALIBRATION
  LSSTCam-imSim/calib/curated                CALIBRATION
LSSTCam-imSim/calib/curated                  CALIBRATION
LSSTCam-imSim/calib/curated/19700101T000000Z RUN
LSSTCam-imSim/calib/unbounded                RUN
LSSTCam-imSim/defaults                       CHAINED
  templates/goodSeeing                       RUN
  skymaps                                    RUN
  LSSTCam-imSim/calib                        CHAINED
    2.2i/calib/gen2                          CALIBRATION
    LSSTCam-imSim/calib/curated              CALIBRATION
  refcats                                    CHAINED
    refcats/cal_ref_cat_2_2                  RUN
  models                                     CHAINED
    pretrained_models/rbResnet50-DC2         RUN
models                                       CHAINED
  pretrained_models/rbResnet50-DC2           RUN
pretrained_models/rbResnet50-DC2             RUN
refcats                                      CHAINED
  refcats/cal_ref_cat_2_2                    RUN
refcats/cal_ref_cat_2_2                      RUN
skymaps                                      RUN
templates/goodSeeing                         RUN
```
Note that [the `ap_verify` spec](https://pipelines.lsst.io/v/daily/modules/lsst.ap.verify/datasets-creation.html#organizing-the-data) _requires_ that `defaults` contain all the "pre-existing" inputs to the pipeline, since that gives the system a way to automatically load the correct data. The intermediate `models` collection (which I forgot to rename after DM-42605) is not strictly necessary, but it makes refreshes, especially partial refreshes, a lot easier.

Reply:
Looks good then.